packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js

```diff
@@ -120,9 +137,10 @@ const withLocalNPMRepo = (func) => {
 const runGenerator = () => {
     // Shelljs can't run interactive programs, so we have to switch to child_process.
     // See https://github.com/shelljs/shelljs/wiki/FAQ#running-interactive-programs-with-exec
-    const cmd = 'npx'
-    const args = ['pwa-kit-create-app', ...process.argv.slice(2)]
-    cp.execFileSync(cmd, args, {stdio: 'inherit'})
+
+    cp.execSync(`npx pwa-kit-create-app --outputDir ${process.argv.slice(3)}`, {
```

Comment:
I know it's not new, but it doesn't seem reasonable to use `npx` for this – it's super easy to accidentally download `pwa-kit-create-app` at a different version from the internet and run that instead. That would be a mistake that would be very hard to spot and correct. 

We want to run a script in this same package on this line. Can we provide the path the the script we want to execute directly instead? Much safer.

Reply:
You're right; I'm wrong. What you're doing here simulates the experience of an end-user installing from real NPM, which is exactly what we want.

Ignore me!!